_docs/style/interactive-states.md

```diff
@@ -0,0 +1,348 @@
+---
+title: Interactive States
+layout: docs
+category: Style
+usage: >
+  The system of interactive states ensures consistent styles for the states of controls in Helix. This system accounts for multiple types of controls, with various interactive patterns, and defines commonalities and standards across all of them.
+
+  All controls that live in the application body strictly adhere to subsets of the specifications in this document, unless otherwise noted in their individual design docs.
+resource: true
+status: complete
+last-modified: 2019-05-29
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+xxxxx.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+xxxxx.
```

Comment:
- In general, only apply hovered states to selectable controls. For example, checkboxes and radios have a hover state, but text inputs do not.
- Do not apply all states styles to your inputs or controls.
- When designing a new control using the interactive states, consult other controls that may be similar to understand which subset of states were used.
- Avoid using selected filled styles for large controls, such as tiles or cards.
- If you have an input or control that is selected and errored, consider whether a better interaction design could have prevented the user from getting into that state.

Reply:
Done.